test/Microsoft.AspNetCore.Rewrite.Tests/project.json

```diff
@@ -5,7 +5,10 @@
   },
   "dependencies": {
     "dotnet-test-xunit": "2.2.0-*",
-    "Microsoft.AspNetCore.Rewrite": "1.0.0-*",
+			"Microsoft.AspNetCore.Rewrite": {
```

Comment:
You may need to rebase: https://github.com/aspnet/BasicMiddleware/commit/ea0042c685ba2228d0d4b85df41fa45a0b360f00